core/conversion/evaluators/aten.cpp

```diff
@@ -426,7 +426,8 @@ auto aten_registrations TRTORCH_UNUSED =
                       }
                     },
                     EvalOptions().validSchemas({
-                        "aten::div.Scalar(Scalar a, Scalar b) -> (float)",
```

Comment:
Did the schemas here change? are these verified function signatures?

Reply:
We should add evaluator tests to make sure these still work correctly

Reply:
Test case is updated